sw/device/silicon_owner/bare_metal/BUILD

```diff
@@ -141,7 +141,7 @@ opentitan_functest(
     key = "multislot",
     ot_flash_binary = ":rom_ext_virtual_bare_metal_virtual",
     signed = True,
-    targets = ["cw310"],
+    targets = ["cw310_rom"],
```

Comment:
Should we delete the `bitstream` definition on line 138?

Reply:
Yes, thanks for catching this, done!